include/gpu_util.h

```diff
@@ -42,11 +42,10 @@ extern void gpu_get_all_domain_id(std::vector<uint16_t>* pgpus_domain_id,
                 std::map<std::pair<uint16_t, uint16_t> , uint16_t>& pgpus_dom_loc_map); 
 extern bool gpu_check_if_mcm_die (int idx);
 extern int gpu_hip_to_smi_index(int hip_index, uint32_t* smi_index);
+extern void gpu_get_all_pci_bdf(std::vector<std::string>* ppci_bdf);
```

Comment:
please use reference instead of pointers , will make it more null safe and general good practice. atleast new APIs can have this way instead of following old ones I feel

Reply:
Yes, will use reference. GPU APIs have used pointers and references already.